riptide-opentelemetry/src/main/java/org/zalando/riptide/opentelemetry/span/StaticSpanDecorator.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.riptide.opentelemetry.span;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Span;
+import org.zalando.riptide.RequestArguments;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StaticSpanDecorator implements SpanDecorator {
+    private final Map<AttributeKey<String>, String> attributes = new HashMap<>();
+
+    public StaticSpanDecorator(Map<String, String> source) {
+        source.forEach((k, v) -> attributes.put(AttributeKey.stringKey(k), v));
```

Comment:
This could result in an NPE. I'm fine, in principle, as this would fail during application startup. Just double checking that this was a conscious decision.

Reply:
my assumption was that as the method isn't annotated as expecting nullable parameter here, it's okay to skip null checks and throw NPE

Reply:
Again, fine by me. My comment was just a safety check.